apps/web/app/(app)/environments/[environmentId]/settings/billing/actions.ts

```diff
@@ -0,0 +1,132 @@
+"use server";
+
+import { authOptions } from "@formbricks/lib/authOptions";
+import { getServerSession } from "next-auth";
+import { AuthorizationError } from "@formbricks/types/errors";
+import { WEBAPP_URL } from "@formbricks/lib/constants";
+import { canUserAccessTeam } from "@formbricks/lib/team/auth";
+import { getTeam } from "@formbricks/lib/team/service";
+import { getMonthlyActivePeopleCount } from "@formbricks/lib/person/service";
+import { getMonthlyDisplayCount } from "@formbricks/lib/display/service";
+
+export async function getBillingDetails(teamId: string) {
+  const session = await getServerSession(authOptions);
+  if (!session) throw new AuthorizationError("Not authorized");
+
+  const isAuthorized = await canUserAccessTeam(session.user.id, teamId);
+  if (!isAuthorized) throw new AuthorizationError("Not authorized");
+
+  const team: any = await getTeam(teamId);
+
+  const stripeCustomerId = team.subscription?.stripeCustomerId;
+
+  if (!stripeCustomerId) {
```

Comment:
why not returning the actual values even though the user has no subscription?

Reply:
this has been handled in the last push now (was not done since initially I was only showing it when a user had purchased one)